### PR TITLE
Reorganize Docs

### PR DESCRIPTION
**Pull request by khajavi**

### CODE REVIEW

**Comment by khajavi** [comment]
@justcoon It is ready for your review!

File: docs/index.md [hunk 1]
```diff
@@ -4,204 +4,119 @@ title: "Introduction to ZIO Logging"
...
+libraryDependencies += "dev.zio" %% "zio-logging-slf4j-bridge" % "@VERSION@"
 
-For example:
+// Scala.js console integration
+libraryDependencies += "dev.zio" %% "zio-logging-jsconsole" % "@VERSION@"
 
-```scala
-myResponseHandler(request) @@ LogAnnotation.UserId(request.userId)
+// Scala.js HTTP Logger which sends logs to a backend via Ajax POST
+libraryDependencies += "dev.zio" %% "zio-logging-jshttp" % "@VERSION@"
```
Comment (justcoon) [inline docs/index.md:46]: zio-logging-jsconsole and zio-logging-jshttp, are zio-logging version 1 only,
they was dropped with version 2

File: docs/logger-setup-in-zio-application.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+```
+
+Expected console output:
+
+```
+timestamp=2022-10-28T18:40:25.517623+02:00 level=INFO thread=zio-fiber-6 message="Start"
+timestamp=2022-10-28T18:40:25.54676+02:00  level=ERROR thread=zio-fiber-0 message="" cause=Exception in thread "zio-fiber-6" java.lang.String: FAILURE
+	at zio.logging.example.SimpleApp.run(SimpleApp.scala:14)
+```
```
Comment (justcoon) [inline docs/logger-setup-in-zio-application.md:36]: this i think is good to move to index.md and not have it like separated page

File: docs/sidebars.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      type: "category",
+      label: "ZIO Logging",
+      collapsed: false,
+      link: { type: "doc", id: "index" },
+      items: [
+        'logger-setup-in-zio-application',
+        'formatting-log-records',
+        'logger-context-and-annotations',
+        'json-console-logger',
+        'colorful-console-logger',
```
Comment (justcoon) [inline docs/sidebars.js:13]: i think we should merge
colorful-console-logger, json-console-logger to one page

console-logger
and structure it in similar way like jpl or slf4j page

have libraryDependencies section, general description, examples ...
to have some consistency in pages

File: docs/index.md [hunk 4]
```diff
@@ -4,204 +4,119 @@ title: "Introduction to ZIO Logging"
...
-package zio.logging.example
+import zio.clock.Clock
+import zio.duration.durationInt
+import zio.logging._
+import zio.random.Random
+import zio.{ExitCode, NonEmptyChunk, ZIO}
 
-import zio.logging.{ LogFormat, console }
-import zio.{ ExitCode, Runtime, Scope, ZIO, ZIOAppArgs, ZIOAppDefault, ZLayer }
+object ZIOLoggingExample extends zio.App {
```
Comment (justcoon) [inline docs/index.md:62]: this look like zio-logging version 1 example

**Review by justcoon — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (khajavi) [reply docs/index.md:46]: So what is the plan for version 1 docs? Do you want to maintain separate documentation for version 1?

File: docs/logger-setup-in-zio-application.md (hunk 2 above)
Comment (khajavi) [reply docs/logger-setup-in-zio-application.md:36]: Currently, we have a (somehow) unique template for the index.md page:

```
# PROJECt_TITLE

DESCRIPTION

PROJECT_BADGES

## Introduction

## Installation

## Example

```

So one example suffices on the index page. Let me know which example is better for that.

File: docs/index.md (hunk 4 above)
Comment (khajavi) [reply docs/index.md]: Done!

File: docs/logger-setup-in-zio-application.md (hunk 2 above)
Comment (khajavi) [reply docs/logger-setup-in-zio-application.md]: I removed the outdated example. Done!

Comment (khajavi) [reply docs/logger-setup-in-zio-application.md]: I removed the outdated example. Done!

File: docs/index.md (hunk 1 above)
Comment (khajavi) [reply docs/index.md]: I removed those from the introduction page. Done!

File: docs/sidebars.js (hunk 3 above)
Comment (khajavi) [reply docs/sidebars.js]: I only merged them. Let's leave the extra works you have mentioned on further PRs.

**Review by justcoon — APPROVED** [review]
@khajavi thank you very much

**Pull request merged**